.github/workflows/test.yml

```diff
@@ -15,28 +15,33 @@ jobs:
     uses: mackerelio/workflows/.github/workflows/go-lint.yml@main
     with:
       os-versions: '["ubuntu-latest", "windows-latest"]'
-  test:
-    uses: mackerelio/workflows/.github/workflows/go-test.yml@main
-    with:
-      os-versions: '["ubuntu-latest", "windows-latest"]'
   test-linux:
-    uses: mackerelio/workflows/.github/workflows/setup-go-matrix.yml@main
+    uses: mackerelio/workflows/.github/workflows/go-test.yml@add-pre-option
```

Comment:
can replace to `main`?

https://github.com/mackerelio/workflows/pull/12 was merged.

Reply:
fixed